src/filtering/markov_kernel.jl

```diff
@@ -201,7 +223,7 @@ function compute_backward_kernel!(
     _matmul!(view(Λ.R, 1:D, 1:D), x.Σ.R, C_DxD)
     # Λ.R[D+1:2D, 1:D] = (G * Q.R')'
     if !isone(diffusion)
```

Comment:
Changing the order could be easier to understand. Same in `predict_cov` I think.